Dockerfile

```diff
@@ -14,8 +14,9 @@ ENV GOPATH /go
 COPY --from=0 /go/ $GOPATH
 ENV PATH $PATH:$GOPATH/bin
 
-RUN apk add --update python python-dev py-pip build-base && \
-    pip install enum34 retrying google-api-core google-cloud-storage && \
+RUN apk add --update python3 python-dev py-pip build-base && \
```

Comment:
py-pip also seems to be 2.7 related, although I cannot really pin point 3.x version for it :/ - https://pkgs.alpinelinux.org/contents?repo=main&page=10&arch=x86&branch=v3.3&name=py-pip

It seems like python3 automatically installs pip3, so it's not needed? When removing this from 2.7 version, Docker fails:

```
RUN apk add --update python && \
    pip install enum34 retrying google-api-core
```

> /bin/sh: pip: not found

but works with python3:
```
RUN apk add --update python3 && \
    pip3 install enum34 retrying google-api-core
```